lib/datasource/hex/index.spec.ts

```diff
@@ -113,5 +126,20 @@ describe(getName(), () => {
       expect(res).toBeDefined();
       expect(httpMock.getTrace()).toMatchSnapshot();
     });
+    it('processes a private repo with auth', async () => {
```

Comment:
```suggestion

    it('processes a private repo with auth', async () => {
```

Add newlines for readabillity. but only for new tests